### DISCUSSION TITLE
How to disable todo.txt.bak file?

### DISCUSSION THREAD

**Discussion by poorlet** [todotxt/todo.txt-cli#327] [Q&A] [answered]
todo.txt.bak file always show up after anything edited with todo.txt-cli, so is there anything config to turn it off, or auto clean it up?

**Comment by inkarkat** [comment, marked as answer]
The backup file is left to be able to manually restore the original file after a mistake (e.g. an accidental deletion of task(s)), or environment problems (like running out of disk space or a corrupted write).

I've given a configuration snippet in #312 to turn this off. What's your motivation for requesting this? Can't you just ignore those backup files (knowning they might one day save you from data loss)?

If there's a valid reason for turning off backups (now that this is the second time I've encountered this request), we could introduce a configuration option (e.g. `TODOTXT_BACKUP_EXT=.bak`) instead of hard-coding it. This would make the reconfiguration both more discoverable (as the variable is mentioned in the verbose help) and simpler (just a one-liner). However, add-ons still have this hard-coded, so it would be a less complete solution than the snippet above.

**Comment by nerdictranslator** [comment]
@inkarkat Okay, it's necromancy time! I understand why you don't want to make a "turn off the backups" option, which is fine. But is there a way to make them show up in my defined TODO_DIR folder or another custom-defined backups folder and not my home folder? I just want them out of the way.

**Reply by inkarkat** [reply]
@nerdictranslator The `sed` function in #312 uses the `/tmp` folder for the backup file, but any file system location could be used for it.
